content/en/docs/tasks/configure-pod-container/translate-compose-kubernetes.md

```diff
@@ -410,7 +410,7 @@ services:
 
 Using `kompose up` with a `build` key:
 
-```sh
```

Comment:
This was just to fix how Atom rendered this code block, and prevent it from erroneously highlighting the whole rest of the file as part of the code block.